app.yaml

```diff
@@ -18,3 +18,7 @@ handlers:
   expiration: 30d
   http_headers:
     Cache-Control: 'public, max-age=2592000'
+
+- url: /_ah/mail/.*
```

Comment:
I don't think you need this. In the main app, it's for incoming emails coming to the app, but what you're doing is handling AJAX through the main controller (above) and _sending_ an email from the backend.

Reply:
Use catchall /.* for main.app instead main_mail.app.